packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -597,6 +597,21 @@ export default class Meeting extends StatelessWebexPlugin {
     this.setUpLocusMediaSharesListener();
     this.setUpLocusInfoMeetingInfoListener();
     this.setUpLocusInfoAssignHostListener();
+    this.mediaInactiveHandler();
+  }
+
+
+  /**
+   * Set up the locus info listener for meetings disconnected due to inactivity
+   * @returns {undefined}
+   * @private
+   * @memberof Meeting
+   */
+  mediaInactiveHandler() {
+    this.locusInfo.on(EVENTS.DISCONNECT_DUE_TO_INACTIVITY, (res) => {
```

Comment:
we should track these type of reconnects as a success and failure in the operational metrics so might be good just to make this location with a TODO